jekyll/_cci2/api-job-trigger.md

```diff
@@ -47,14 +47,12 @@ For a complete reference of the API,
 see the [CircleCI API Documentation]({{ site.baseurl }}/api/v1-reference/).
 
 **Note:**
-While you can **not** trigger [workflows]({{ site.baseurl }}/2.0/workflows/) with the CircleCI API,
-the `config.yml` that contains the job definition
-can still have a `workflows` section.
+It is possible to trigger [workflows]({{ site.baseurl }}/2.0/workflows/) with the CircleCI API, using a new endpoint, see the Trigger a Build with a Workflow section of the [CircleCI API Projects Documentation]({{ site.baseurl }}/api/v1-reference/#projects).
+
+Jobs triggered with the API may contain a `workflows` section.
 These workflows do **not** have to reference the job
 you trigger with the API.
-
-Jobs triggered with the API
-do **not** have access to environment variables
+Jobs triggered with the API do **not** have access to environment variables
```

Comment:
I'm not sure this is true with the new endpoint - may need to check with engineering on this.

Reply:
Ok, I didn't add this, it was there already and it is relatively new, so I will leave it